batch/sql/estimated-current.sql

```diff
@@ -595,101 +595,43 @@ BEGIN
   SET cur_billing_date = CAST(UTC_DATE() AS DATE);
 
   IF msec_diff_rollup != 0 THEN
-    INSERT INTO aggregated_billing_project_user_resources_v2 (billing_project, user, resource_id, token, `usage`)
-    SELECT billing_project, `user`,
-      resource_id,
-      rand_token,
-      msec_diff_rollup * quantity
-    FROM attempt_resources
-    JOIN batches ON batches.id = attempt_resources.batch_id
-    WHERE batch_id = NEW.batch_id AND job_id = NEW.job_id AND attempt_id = NEW.attempt_id
-    ON DUPLICATE KEY UPDATE `usage` = `usage` + msec_diff_rollup * quantity;
-
     INSERT INTO aggregated_billing_project_user_resources_v3 (billing_project, user, resource_id, token, `usage`)
     SELECT batches.billing_project, batches.`user`,
-      attempt_resources.deduped_resource_id,
+      attempt_resources.resource_id,
```

Comment:
Can you remind me why this column name is changing?

Reply:
At this point `resource_id` **should** always equal `deduped_resource_id` with the previous fixes to how we insert resources to not be duplicated. I made this change here in case we do decide to swap the columns "deduped_resource_id" -> "resource_id" and drop the original resource id column (has duplicates in it). We can save that for a separate step if that's easier.

Reply:
Ok, I think that checks out but ya I think I works prefer just doing that separately if that's alright.